src/docker.py

```diff
@@ -57,6 +57,8 @@ def extract_resources_from_docker(self):
             self.__extract_from_docker('crustio/crust', '/opt/crust/crust')
         elif self.chain_name == 'subsocial':
             self.__extract_from_docker('dappforce/subsocial-parachain', '/usr/local/bin/subsocial-collator')
+        elif self.chain_name == 'robonomics':
+            self.__extract_from_docker('robonomics/robonomics', 'usr/local/bin/robonomics')
```

Comment:
Can you please rebase on `origin/develop`?

This code should already be there, but `main` wasn't merged into `develop` last revision release. Just fixed that.